page/GoalSettingPage.js

```diff
@@ -0,0 +1,12 @@
+var goalSetting_Page = function() {
+    title= element(by.css('h1.clp-lead__title'))
+    lectures=element(by.css('.num-lectures'))
+
+    this.getTitle = function () {
```

Comment:
@vinaygadegela Is this working  without using  this keyword here

Reply:
yes